Config/SolutionSettingsTests.cs

```diff
@@ -0,0 +1,118 @@
+        private const int TIMEOUT_MS = 10000;
+        private Dictionary<string, string> disposeList = new Dictionary<string, string>();
+        private const string COLLECTION_ID = "/solution-settings";
+        private const string THEME_KEY = "/theme";
+        private const string LOGO_KEY = "/logo";
+        private const string AZURE_MAPS_KEY = "AzureMapsKey";
+        private const string TEST_JSON = @"{
+                                        testObject: '123456789',
+                                        Type: 'application/json'
+                                        }";
+
+        public SolutionSettingsTests()
+        {
+            this.httpClient = new HttpClient();
+        }
+
+        /// <summary>
+        /// Test that the service can set a solution-settings file in CosmosDB
+        /// </summary>
+        [Fact, Trait(Constants.TEST, Constants.INTEGRATION_TEST)]
+        public void ShouldSetSolutionSettingsTheme()
+        {
+            var request = new HttpRequest();
+            request.Options.Timeout = TIMEOUT_MS;
+            request.SetUriFromString(Constants.CONFIG_ADDRESS + COLLECTION_ID + THEME_KEY);
+            request.SetContent(TEST_JSON);
+
+            // Act for update
+            var response = this.httpClient.PutAsync(request).Result;
+
+            // Assert for update
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+            var themeOut = JsonConvert.DeserializeObject(response.Content) as JToken ?? new JObject();
+            Assert.NotNull(themeOut[AZURE_MAPS_KEY]);
+        }
+
+        /// <summary>
+        /// Test that the service can get a solution-settings file in CosmosDB
+        /// </summary>
+        [Fact, Trait(Constants.TEST, Constants.INTEGRATION_TEST)]
+        public void ShouldGetSolutionSettingsTheme()
+        {
+            // Arrange
+            this.ShouldSetSolutionSettingsTheme();
+            var request = new HttpRequest();
+            request.Options.Timeout = TIMEOUT_MS;
+            request.SetUriFromString(Constants.CONFIG_ADDRESS + COLLECTION_ID + THEME_KEY);
+
+            // Act
+            var response = this.httpClient.GetAsync(request).Result;
+
+            // Assert
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+            var themeOut = JsonConvert.DeserializeObject(response.Content) as JToken ?? new JObject();
+            Assert.NotNull(themeOut[AZURE_MAPS_KEY]);
+        }
+
+        /// <summary>
+        /// Test that the service can set a solution-settings file in CosmosDB
+        /// </summary>
+        [Fact, Trait(Constants.TEST, Constants.INTEGRATION_TEST)]
+        public void ShouldSetSolutionSettingsLogo()
+        {
+            var request = new HttpRequest();
+            request.Options.Timeout = TIMEOUT_MS;
+            request.SetUriFromString(Constants.CONFIG_ADDRESS + COLLECTION_ID + LOGO_KEY);
+            request.SetContent(TEST_JSON);
```

Comment:
logo isn't uploaded as json, it's uploaded as an image binary

Reply:
done